alphazero/_game.py

```diff
@@ -228,11 +229,11 @@ def current_legal_moves(self) -> torch.Tensor:
 
     @staticmethod
     def get_game_status(board) -> tuple:
-        verts_0 = any((board == 0).sum(axis=0) == 3)
-        horizontals_0 = any((board == 0).sum(axis=1) == 3)
+        verts_0 = any((board == 0).sum(dim=0) == 3)
+        horizontals_0 = any((board == 0).sum(dim=1) == 3)
 
-        verts_1 = any((board == 1).sum(axis=0) == 3)
-        horizontals_1 = any((board == 1).sum(axis=1) == 3)
+        verts_1 = any((board == 1).sum(dim=0) == 3)
+        horizontals_1 = any((board == 1).sum(dim=1) == 3)
```

Comment:
is this one of those deprecated kwargs things that pytorch didn't warn me about?

Reply:
Could be, torch generally uses dim instead of axis, at least in newer versions